apps/src/templates/certificates/PetitionCallToAction.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PetitionForm from './PetitionForm';
+
+export default function PetitionCallToAction() {
+  return (
+    <>
+      <div id="petition-block" className="petition_block">
```

Comment:
minor style thing, but prefer to use snake-case instead of camel_case for css class names.

Reply:
Ahh, thanks!